### PR TITLE
Feature/add new cookie banner

### PR DESCRIPTION
**Pull request by midudev**
New cookie banner component based on the one being used by Fotocasa.

image

### CODE REVIEW

**Review by FranDelaCasa — APPROVED** [review]

File: components/cookie/banner/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run babel && npm run sass",
+    "babel": "../../../node_modules/.bin/babel --presets schibsted-spain ./src --out-dir ./lib",
+    "sass": "../../../node_modules/.bin/cpx \"./src/**/*.scss\" ./lib",
+    "preversion": "echo \"preversion\"",
+    "version": "npm run build",
+    "postversion": "git add ${PWD} && git commit -m \"release(cookie/banner): v$(node -p -e \"require('./package.json').version\")\" && git push origin master && npm publish --access=public"
```
Comment (JoanClaret) [inline components/cookie/banner/package.json]: `preversion`, `version` and `postversion` is not needed anymore with suistudio v4.

File: components/cookie/banner/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@schibstedspain/suistudio-fatigue-deps": "github:SUI-Components/suistudio-fatigue-deps",
```
Comment (JoanClaret) [inline components/cookie/banner/package.json]: what about using `@schibstedspain/sui-component-dependencies` ?
`suistudio-fatigue-deps` is deprecated.

**Review by JoanClaret — APPROVED** [review]
2 minor comments, 
great job anyways!

File: components/cookie/banner/package.json (hunk 2 above)
Comment (midudev) [reply components/cookie/banner/package.json]: You're right, my bad as this component was created with the previous version and while I was working on it, it got updated. Thanks for your 👀

**Comment by midudev** [comment]
Addressed feedback @JoanClaret Thanks! 🙇

File: components/cookie/banner/package.json (hunk 2 above)
Comment (JoanClaret) [reply components/cookie/banner/package.json]: The same happened to me in my last component. No worries!
; )

**Pull request merged**